flink-table/flink-table-common/src/main/java/org/apache/flink/table/expressions/CallExpression.java

```diff
@@ -57,13 +60,13 @@
     private final DataType dataType;
 
     public CallExpression(
-            FunctionIdentifier functionIdentifier,
+            boolean isTemporary,
```

Comment:
why don't create similar methods as `ContextResolvedFunction` so we callers no need to pass true/false and nulls?

Reply:
I updated `CallExpression`